--- segment-duration.go.orig	2021-02-27 21:24:25.000000000 -0500
+++ segment-duration.go	2021-02-27 21:25:59.000000000 -0500
@@ -85,10 +85,7 @@
 	} else if ns > milliseconds {
 		millis := ns / milliseconds
 		ns -= millis * milliseconds
-		micros := ns / microseconds
-		content = fmt.Sprintf("%dms %d\u00B5s", millis, micros)
-	} else {
-		content = fmt.Sprintf("%d\u00B5s", ns/microseconds)
+		content = fmt.Sprintf("%dms", millis)
 	}

 	return []pwl.Segment{{
